src/aleph_pytezos/crypto/key.py

```diff
@@ -126,12 +129,13 @@ def from_secret_exponent(
         """
         # Ed25519
         if curve == b'ed':
-            raise Exception("Curve ed25519 not supported.")
             # Dealing with secret exponent or seed?
-            #if len(secret_exponent) == 64:
-            #    public_point = pysodium.crypto_sign_sk_to_pk(sk=secret_exponent)
-            #else:
-            #    public_point, secret_exponent = pysodium.crypto_sign_seed_keypair(seed=secret_exponent)
+            if len(secret_exponent) == 64:
+                public_point = nacl.signing.SigningKey(secret_exponent[32:]).verify_key.encode()
```

Comment:
Why this `[32:]` ? Can this be documented in a comment ?

Reply:
See below, it is how pynacl treats Ed25519 secret keys. When generating a secret key for i.e. Solana, you will get the same format of 64 bytes.

Reply:
But, to be honest, it could be nicer if
`nacl.signer.VerifyKey(secret_exponent[:32]).encode()`

Reply:
Would something like this be more explicit then ?

```suggestion
                # The standard format of a secret key is 64 bytes, the first 32 bytes are the public key, the last 32 are secret.
                public_key, secret = secret_exponent[:32], secret_exponent[32:]
                public_point = nacl.signing.SigningKey(secret).verify_key.encode()
```